module_fcst_grid_comp.F90

```diff
@@ -560,6 +574,12 @@ subroutine fcst_initialize(fcst_comp, importState, exportState, clock, rc)
 !       if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
 !     endif
 !
+!test to write out netcdf file:
+        if (btest(diagnostic,16)) then
```

Comment:
What does this line do? Can we use a flag to control if we want to output the grid file?

Reply:
I use it to write out the regional grid information if bit 16 is set. The function, which I stored with io writes out a netcdf file with center coordinates, corner coordinates, mask, area computed by regridding (when needed), and area stored in grid. I don't know if this will be needed going forward but it help me diagnose the stagger alignment needed for the corners.

Reply:
This is to cryptic, and non intuitive. Can't you use more descriptive name, like 'diagnose_grid_info' and make it logical variable. I do not see any advantage of using bit 16 in an integer to store information that user wants this debug file to be written out.
I know you can store up to 32 bits of information in a single variable, but that just makes configuration file less readable, documentation more complicated etc.

Reply:
This comment has been addressed in https://github.com/NOAA-EMC/fv3atm/pull/355/commits/6ac48b1d36b5559500dcb4718bae9140206fc2b8
diagnostic was removed and grid write to netcdf now uses cplprint_flag